spring-ai-core/src/main/java/org/springframework/ai/chat/model/MessageAggregator.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.springframework.ai.chat.service;
+package org.springframework.ai.chat.model;
```

Comment:
I think I had moved this out of the model package to avoid a dependency cycle, will need to revisit.